src/keyDerivation_test.c

```diff
@@ -100,9 +104,13 @@ void testcase_derivePublicKey(uint32_t* path, uint32_t pathLen, const char* expe
 	PRINTF_PATH(path, pathLen);
 	PRINTF("\n");
 
+	path_spec_t pathSpec;
+	pathSpec.length = pathLen;
+	os_memmove(pathSpec.path, path, MAX_PATH_LENGTH * 4);
```

Comment:
Again same argument. Also, maybe abstract this into "pathSpec_init(spec*, path, pathLen)` ?